src/db/mongo/repo.js

```diff
@@ -15,6 +15,11 @@ exports.getRepo = async (name) => {
   return collection.findOne({ name: { $eq: name } });
 };
 
+exports.getRepoByUrl = async (url) => {
+  const collection = await connect(cnName);
+  return collection.findOne({ url: { $eq: url}});
```

Comment:
`return collection.findOne({ org: ..., repo: ..., platform: ... })`

Thoughts? We should probably step back a little and define a data model for repository together that will serve us well moving forward and isn't too opinionated. I am not sure about URL as only as this is quite un-atomic in its nature and includes content that we don't need in DB, i.e. `https`, `://` - I feel like it can be broken down but happy to hear otherwise of course.